### PR TITLE
use classes/const/let etc

### PR DESCRIPTION
**Pull request by jimmywarting**
- [x] Have you updated CHANGELOG.md? yes

This conversion was done using lebab and `standard --fix`

```
lebab --replace <FILE> --transform arrow,let,includes,template,default-param,obj-shorthand,obj-method,arg-spread,arg-rest,for-each,class
```

wanted to include ESM but maybe to much breaking...

### CODE REVIEW

**Comment by jimmywarting** [comment]
question: why do only some file lint with standardjs and not all?

File: lib/DOMMatrix.js [hunk 1]
```diff
@@ -63,68 +61,441 @@ function parseTransform(tform) {
...
+  toFloat32Array() {
+    return Float32Array.from(this._values)
+  }
+
+  toFloat64Array() {
+    return this._values.slice(0)
   }
 }
 
-DOMMatrix.fromMatrix = function (init) {
```
Comment (zbjornson) [inline lib/DOMMatrix.js:119]: Could you make these static members maybe? `static fromMatrix(init) { ... }`

**Review by zbjornson — COMMENTED** [review]
This looks great, thank you. One suggestion to keep using modern syntax.

> why do only some file lint with standardjs and not all?

This was an attempt to gradually update the codebase to follow StandardJS formatting, but it's been so slow that it's created a lot of confusion. We should just convert everything finally.

File: lib/DOMMatrix.js (hunk 1 above)
Comment (jimmywarting) [reply lib/DOMMatrix.js:119]: will do

**Comment by jimmywarting** [comment]
bdabed4 might be a bit have to look through... basically executed `standard --fix` on the hole project and fixed some error's manually

**Comment by jimmywarting** [comment]
pump

**Comment by jimmywarting** [comment]
bump

**Review by zbjornson — APPROVED** [review]
Thanks much! Sorry for taking forever to land this.

I just rebased to fix the merge conflicts and squashed the commits that were related.

**Pull request merged**